packages-next/wysiwyg/src/views/index.tsx

```diff
@@ -0,0 +1,46 @@
+/* @jsx jsx */
+
+import { jsx } from '@keystone-ui/core';
+import {
+  CardValueComponent,
+  CellComponent,
+  FieldController,
+  FieldControllerConfig,
+} from '@keystone-next/types';
+import { FieldContainer, FieldLabel } from '@keystone-ui/fields';
+
+export { Field } from './Field';
+
+export const Cell: CellComponent = ({ item, field }) => {
+  const data = item[field.path];
+  if (!data) return null;
+  return (data);
+};
+
+export const CardValue: CardValueComponent = ({ item, field }) => {
```

Comment:
have not tested this yet. hopefully we can have a view only field and then an edit button to render tiny editor